script/deploy/safe/confirm-safe-tx.ts

```diff
@@ -1,173 +1,61 @@
+/**
+ * Confirm Safe Transactions
```

Comment:
When I sign a transaction it's still showing that I can sign again. Once the transaction it's signed for specific address shouldn't it disappear?

Reply:
Are you sure? Can you post a screenshot? I just tried and the only option it gives me is "Do Nothing"

Reply:
Maybe it's because of RPC issue in with different network? 

https://gist.github.com/mirooon/937947f1fae08268100cf35a1304f007